README.Rmd

```diff
@@ -154,30 +144,33 @@ We start by creating a new factory in the temporary folder:
 ```{r creation}
 library(reportfactory)
 
-destination <- file.path(tempdir(), "new_factory")
+destination <- file.path(tempdir(), "new_factory") # In practise, one should not use a temporary directory else all will be lost.
 destination
 new_factory(destination)
 dir()
 
 ```
 
-By default, examples of reports (using simulated epidemiological data) are added
-to the factory; these can be listed by:
+By default, examples of reports (using simulated epidemiological data) are added to the factory; these can be listed by:
 
 ```{r listing, echo = -1}
 setwd(destination)
 list_reports()
 list_outputs()
 list_deps() # list all needed packages
 list_deps(missing = TRUE) # list only missing ones
+
+fs::dir_tree() # Clarity - show directory tree
 ```
 
 To compile a single report, one can use:
 
 ```{r compile_one, echo = -1}
 setwd(destination)
-compile_report("contacts_2017-10-29", quiet = TRUE)
+compile_report("example_report_2019-01-31.Rmd", quiet = TRUE)
 list_outputs()
+
+fs::dir_tree() # Clarity - show how directory tree changes after compilation
```

Comment:
Use base R, a package in the deps, or add {fs} to DESCRIPTION Suggests

Reply:
✅